README.md

```diff
@@ -478,6 +490,7 @@ spec:
 Depending on the type of tasks that you'll be running, you may find that you need the task pod to run as `privileged`. This can open yourself up to a variety of security concerns, so you should be aware (and verify that you have the privileges) to do this if necessary. In order to toggle this feature, you can add the following to your custom resource:
 
 ```yaml
+# ansible/instantiate-awx-deployment.yml
```

Comment:
I do not think repeating this over and over provides any value. Plus, the file you reference here is a playbook rather than an easily-copyable Kubernetes object. This is more likely to cause confusion than it is to help. Please drop all of the `# ansible/instantiate-awx-deployment.yml` comments.